### PR TITLE
*: Cut v0.16.0-alpha.0

### PR DESCRIPTION
**Pull request by mxinden**
Given the long list of changes I would like to start with an alpha release first. Hence this replaces https://github.com/prometheus/alertmanager/pull/1553.

Signed-off-by: Max Leonard Inden <IndenML@gmail.com>

### CODE REVIEW

**Comment by mxinden** [comment]
@stuartnelson3 do you think we should hold here due to https://github.com/prometheus/alertmanager/issues/1643? I would be fine with a patch release once we can reproduce the issue.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,45 @@
...
 * [CHANGE] Revert Alertmanager working directory changes in Docker image back to `/alertmanager` (#1435)
-* [CHANGE] Remove `api/v1/alerts/groups` GET endpoint (#1508)
+* [CHANGE] Using the recommended label syntax for maintainer in Dockerfile (#1533)
+* [CHANGE] Change `alertmanager_notifications_total` to count attempted notifications, not only successful ones (#1578)
+* [CHANGE] Run as nobody inside container (#1586)
+* [CHANGE] Support `w` for weeks when creating silences, remove `y` for year (#1620)
+* [FEATURE] Introduce OpenAPI generated API v2 (#1352)
+* [FEATURE] Lookup parts in strings using regexp.MatchString in templates (#1452)
+* [FEATURE] Support image/thumb url in attachment in Slack notifier (#1506)
+* [FEATURE] Support custom TLS certificates (#1528)
```
Comment (simonpasquier) [inline CHANGELOG.md]: `Support custom TLS certificates for the email notifier (#1528)`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,45 @@
...
+* [FEATURE] [amtool] Add timeout support to amtool commands (#1471)
 * [FEATURE] [amtool] Added `config routes` tools for vizualization and testing routes (#1511)
+* [FEATURE] [amtool] Support adding alerts using amtool (#1461)
+* [ENHANCEMENT] Add help link in UI to Alertmanager documentation (#1522)
+* [ENHANCEMENT] Enforce HTTP or HTTPS URLs in Alertmanager config (#1567)
+* [ENHANCEMENT] Make OpsGenie API Key a templated string (#1594)
+* [ENHANCEMENT] Add name, value and SlackConfirmationField to action in Slack notifier (#1557)
+* [ENHANCEMENT] Show more alert information on silence form and silence view pages (#1601)
+* [ENHANCEMENT] Add cluster peers DNS refresh job (#1428)
+* [ENHANCEMENT] [amtool] Support assuming first label is alertname in alert query with matchers (#1575)
```
Comment (simonpasquier) [inline CHANGELOG.md]: more a bugfix?

**Comment by mxinden** [comment]
> Just a couple of nits.

Thanks @simonpasquier. Fixed.

**Comment by stuartnelson3** [comment]
Looks good once it's green

**Pull request merged**